install/martypc.toml

```diff
@@ -152,14 +152,14 @@ video = "CGA"
 # "None"  - No hard disk controller will be present
 # "Xebec" - Emulates the IBM/Xebec 20MB Fixed Disk Controller
 
-hdc = "None"
-#hdc = "Xebec"
+#hdc = "None"
+hdc = "Xebec"
 
 # VHD to mount into drive0 (Typically C:)
-#drive0 = "dos330.vhd"
+drive0 = "dos330.vhd"
```

Comment:
there is no point telling the emulator to load VHD files that the user won't have